hack/examples/golang/callfunction/fibonacci/fibonacci.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2017 The Nuclio Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"math"
+	"strconv"
+
+	"github.com/nuclio/nuclio/pkg/errors"
+
+	"github.com/nuclio/nuclio-sdk-go"
+)
+
+func fib(n uint64) (uint64, error) {
+	if n == 0 {
+		return 0, nil
+	}
+	var a, b uint64 = 0, 1
+	for i := uint64(0); i < n-1; i++ {
+		if a < math.MaxUint64-b {
+			a, b = b, a+b
+		} else {
+			return 0, errors.New("Overflow. Too big request")
+		}
+	}
+	return b, nil
+}
+
+func Handler(context *nuclio.Context, event nuclio.Event) (interface{}, error) {
```

Comment:
Style:
1. Public first, then private (move Handler up top)
2. Use InfoWith, not Info